<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1052</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>CADASTRO</string>
     </attribute>
     <widget class="QPushButton" name="atualizar_button">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>140</width>
        <height>42</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0 ,0, 0);
background-color: rgb(255, 204, 0);</string>
      </property>
      <property name="text">
       <string>ATUALIZAR</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="parceiro_input">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>120</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(254, 254, 254);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="status_comboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>431</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(254, 254, 254);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="resumo_input">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>431</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(254, 254, 254);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pesquisar_button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>320</y>
        <width>140</width>
        <height>42</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 102, 255);</string>
      </property>
      <property name="text">
       <string>PESQUISAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletar_button">
      <property name="geometry">
       <rect>
        <x>406</x>
        <y>400</y>
        <width>140</width>
        <height>42</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>DELETAR</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="demanda_input">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(254, 254, 254);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="qtd_total_label">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>230</y>
        <width>200</width>
        <height>42</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(254, 254, 254);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(254, 43, 52, 255), stop:0.5 rgba(158, 25, 184, 255), stop:1 rgba(101, 0, 254, 255));
font: bold;</string>
      </property>
      <property name="text">
       <string>  TOTAL: </string>
      </property>
     </widget>
     <widget class="QLabel" name="qtd_aguardando_usuario_label">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>350</y>
        <width>411</width>
        <height>42</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(254, 254, 254);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(254, 43, 52, 255), stop:0.5 rgba(158, 25, 184, 255), stop:1 rgba(101, 0, 254, 255));
font: bold;</string>
      </property>
      <property name="text">
       <string>  AGUADANDO O USUARIO: </string>
      </property>
     </widget>
     <widget class="QLabel" name="qtd_atendimento_label">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>290</y>
        <width>411</width>
        <height>42</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(254, 254, 254);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(254, 43, 52, 255), stop:0.5 rgba(158, 25, 184, 255), stop:1 rgba(101, 0, 254, 255));
font: bold;</string>
      </property>
      <property name="text">
       <string>  EM ATENDIMENTO: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="limpar_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>140</width>
        <height>42</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(153, 153, 153);</string>
      </property>
      <property name="text">
       <string>LIMPAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="status_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
     <widget class="QLabel" name="qtd_aguardando_parceiro_label">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>410</y>
        <width>411</width>
        <height>42</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(254, 254, 254);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(254, 43, 52, 255), stop:0.5 rgba(158, 25, 184, 255), stop:1 rgba(101, 0, 254, 255));
font: bold;</string>
      </property>
      <property name="text">
       <string>  AGUADANDO PARCEIRO: </string>
      </property>
     </widget>
     <widget class="QLabel" name="parceiro_label">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Parceiro</string>
      </property>
     </widget>
     <widget class="QLabel" name="resumo_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Resumo</string>
      </property>
     </widget>
     <widget class="QLabel" name="qtd_fechada_label">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>470</y>
        <width>230</width>
        <height>42</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(254, 254, 254);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(254, 43, 52, 255), stop:0.5 rgba(158, 25, 184, 255), stop:1 rgba(101, 0, 254, 255));
font: bold;</string>
      </property>
      <property name="text">
       <string>  FECHADAS: </string>
      </property>
     </widget>
     <widget class="QLabel" name="demanda_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Demanda</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cadastrar_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>140</width>
        <height>42</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 179, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>CADASTRAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="titulo_label_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1051</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(254, 43, 52, 255), stop:0.5 rgba(158, 25, 184, 255), stop:1 rgba(101, 0, 254, 255));
font: bold;
</string>
      </property>
      <property name="text">
       <string>Cadastro de Demandas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>60</y>
        <width>181</width>
        <height>184</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/tinywow_aad1d1b6ed3aa282902ff63368cbeedc_4600137.jpg</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>LISTA</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>111</y>
        <width>1051</width>
        <height>551</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>38</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </row>
      <column>
       <property name="text">
        <string>DEMANDA</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STATUS</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RESUMO</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PARCEIRO</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DATA</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>2022-12345678</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>STATUS</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>RESUMO</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>PARCEIRO</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>00/00/0000</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="titulo_label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1051</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(254, 43, 52, 255), stop:0.5 rgba(158, 25, 184, 255), stop:1 rgba(101, 0, 254, 255));
font: bold;
</string>
      </property>
      <property name="text">
       <string>Lista de Demandas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="atualizar_button_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>281</width>
        <height>42</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(153, 153, 153);
</string>
      </property>
      <property name="text">
       <string>ATUALIZAR LISTAGEM</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
